docs/Python-API.md

```diff
@@ -12,7 +12,7 @@ from unityagents import UnityEnvironment
 env = UnityEnvironment(file_name=filename, worker_id=0)
 ```
 
-* `file_name` is the name of the environment binary (located in the root directory of the python project). 
+* `file_name` is the name of the environment binary (located in the root directory of the python project), remember to wrap the `file_name` with `""`. 
```

Comment:
I would say something more like : It is of type string. I think it not necessary